charts/cluster-scanner/values.yaml

```diff
@@ -104,7 +104,7 @@ runtimeStatusIntegrator:
     # The image repository to use for pulling the Runtime Status Integrator
     # image
     repository: sysdig/runtime-status-integrator
-    tag: "0.8.3"
+    tag: 0.9.0
```

Comment:
It seems that the automated PR removes the surrounding " from the tag. Is this intended?